server/ngwmn/services/ngwmn.py

```diff
@@ -188,83 +188,147 @@ def get_well_log(agency_cd, location_id):
     if water_well is None:
         return {}
 
-    return {
+    result = {
```

Comment:
I changed this because some of the site's return a very sparse well log document. I took a conservative approach and checked every tag for existence before processing further.